### DISCUSSION TITLE
compatible camera

### DISCUSSION THREAD

**Discussion by kmchen13** [Motion-Project/motion#1791] [General]
Hi, 

I failed to configure completely an EZVIZ TY1 camera with motion on Debian11. It seems to be locked by proprietary software. I got to configure it and manage it through local network but failed through WAN/Fast 5657 Sagemcom router.

I found a "compatible devices list" at https://www.lavrsen.dk/foswiki/bin/view/Motion/WorkingDevices but it ends on 2017.

Is there some updated list anywhere or could someone tell some contemporary supported network cameras with sufficient local storage ?

Any help greatly appreciated

**Comment by Mr-Dave** [comment]
Since there are literally millions of different brands and camera models being sold (and it changes by the minute), creating any such list is a futile effort.  

Instead look for camera functionality.  Specifically supported camera protocol and things listed such as RTSP,RTMP, ONVIF, etc.  If the camera lists RTSP, I'd say that that it is very very likely that it would work with Motion/Motionplus.  (I'd like to say it is 100% but I'm sure  some camera somewhere out of the millions won't work.  )

Other red flags on the cameras is what you've already encountered. Proprietary software.  If there is any sort of "additional plan" or "service" offered by the vendor, it likely will not work with Motion/Motionplus.  

Finally, you've mentioned camera storage.  When using Motion/Motionplus, the camera storage can not be used at all.  The software only monitors the RTSP feed from the camera camera and stores information on devices connected to the computer running Motion/Motionplus.

**Comment by kmchen13** [comment]
Thanks for your answer. Yes this camera is RTSP compatible and yes it is under proprietary software. So I'm stucked.

Can someone mention a camera he knows that is working under Motion software and still on the market ?

**Reply by tosiara** [reply]
This has been asked many times before, you can check the mail archive, ex: https://sourceforge.net/p/motion/mailman/message/58715183/

Algorithm should be the following:
1. Go to any online web shop that delivers to your location
2. Filter cameras by needed specs (indoor/outdoor, IR, resolution etc)
3. Read description
4. If RTSP is mentioned in specs and not mentioned mandatory mobile app - make an order
5. When received the order check if it really works without a mobile app. If it requires an app - return the order.
6. Follow the instruction manual to turn on and configure the camera
7. If the camera still does not work with motion - post an issue on Github

If you get chinese camera, don't forget to put it on a vlan without internet access
